src/test/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptorTest.java

```diff
@@ -538,6 +538,27 @@ public void testEnableDisableConflict() {
                 "--" + ReadFilterArgumentDefinitions.DISABLE_READ_FILTER_LONG_NAME, "GoodCigarReadFilter"});
     }
 
+    @Test(expectedExceptions = CommandLineException.class)
```

Comment:
I might either add a new more specific exception type or scrape the message.  It's really easy to tests like this failing for other reasons like typos and not the one you expect.

Reply:
idk... commandline is a very specfic exception and i'm not super inclined to split the exception up into dozens of hyper specific commandline exceptions. Furthermore.... if you look up in the class this is already all over the place in these tests. No action

Reply:
I'm in favor of hyper specific exceptions all over the place but that's fine.  Did you check that it's actually erroring for the reasons you expect?  I says this from experience constructing commandline tests that turned out to be typos and not valid tests.